website/docs/r/event_orchestration_unrouted.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_event_orchestration_unrouted"
+sidebar_current: "docs-pagerduty-resource-event-orchestration-unrouted"
+description: |-
+  Creates and manages an Unrouted Orchestration for a Global Event Orchestration in PagerDuty.
+---
+
+# pagerduty_event_orchestration_unrouted
+
+An Unrouted Orchestration allows users to create a set of Event Rules that will be evaluated against all events that don't match any rules in the Orchestration's Router.
+
+The Unrouted Orchestration evaluates events sent to it against each of its rules, beginning with the rules in the "start" set. When a matching rule is found, it can modify and enhance the event and can route the event to another set of rules within this Unrouted Orchestration for further processing.
+
+## Example of configuring a Unrouted Rules for an Orchestration
+
+  In this example the user has defined the unrouted orchestration with two rules, each routing to a different service.This example assumes services used in the route_to configuration already exists. So it does not show creation of service resource.As there is a single set, route_to is not defined for the rules.
+
+  If there are more than one set, the rules in start set must define route_to with id of the next set
+  The first rule matches only if the condition is matched. The second rule matches always as there are no conditions.
```

Comment:
**Not a Blocker:** Since this example doesn't demonstrate nested rules, I don't think we need to mention nested rules in this sentence specifically. Maybe something like this instead?

```suggestion
  In this example of an Unrouted Orchestration, first rule matches only if the condition is matched. The second rule matches all remaining events because it doesn't have any conditions.
```